Lib/Core/IndyCore290.rc

```diff
@@ -0,0 +1,31 @@
+1 VERSIONINFO 
+FILEVERSION 10,6,2,0
+PRODUCTVERSION 10,6,2,0
+FILEFLAGSMASK 0x3FL
+FILEFLAGS 0x00L
+FILEOS 0x40004L
+FILETYPE 0x1L
+FILESUBTYPE 0x0L
+{
+ BLOCK "StringFileInfo"
+ {
+  BLOCK "000104E4"
+  {
+   VALUE "CompanyName", "Chad Z. Hower a.k.a Kudzu and the Indy Pit Crew\0"
+   VALUE "FileDescription", "Internet Direct (Indy) 10.6.2 - Core Run-Time Package\0"
+   VALUE "FileVersion", "10.6.2.0\0"
+   VALUE "InternalName", "IndyCore280\0"
```

Comment:
Shouldn't it be 290 instead? And few more below.

Reply:
Yes, 290.